docs/src/rules/no-invalid-this.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: no-invalid-this
 rule_type: suggestion
+handled_by_typescript: true
+extra_typescript_info: Note that, technically, TypeScript will only catch this if you have the `strict` or `noImplicitThis` flags enabled. These are enabled in most TypeScript projects, since they are considered to be best practice.
```

Comment:
```suggestion
extra_typescript_info: Note that, technically, TypeScript will only catch this if you have the `strict` or `noImplicitThis` flags enabled. These are enabled in most TypeScript projects since they are considered to be best practices.
```

Reply:
its written as non-plural because strict is a superset containing noimplicitthis, so strict is best practice and noimplicitthis is not